code/game/machinery/portable_turret.dm

```diff
@@ -902,7 +902,7 @@ Status: []<BR>"},
 
 /obj/machinery/porta_turret_cover/attack_hand(mob/user)
 	. = ..()
-	if(.)
+	if(. || !Parent_Turret)

```

Comment:
I guess the solution to that is to add a check in New() then GC it if it can't find a parent?